tests/Api/Category/GetCategoryIntegration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Akeneo\Pim\tests\Api\Category;
+
+use Akeneo\Pim\tests\Api\ApiTestCase;
+
+class GetCategoryIntegration extends ApiTestCase
+{
+    public function testGet()
+    {
+        $api = $this->createClient()->getCategoryApi();
+
+        $category = $api->get('sandals');
+
+        $this->assertInternalType('array', $category);
```

Comment:
It's mostly to have a assertion fail instead of a php error if the response is not an array.